module/VuFind/tests/fixtures/misc/testbug2.json

```diff
@@ -59,7 +59,7 @@
           "Vico, Giambattista, 1668-1744.",
           "La congiura dei Principi Napoletani 1701 : (prima e seconda stesura) /"],
         "isbn":[
-          "8820737493"],
+          "8820737493", "9788072815562"],
```

Comment:
Might be worth also adding a 979 ISBN (one that can't be converted to ISBN-10, e.g. 9791032917633) to the test fixture so that some of the flags can be more thoroughly tested. I'd recommend making the 979 ISBN first, so we can ensure that prefer10 is actually doing something.

Reply:
Adding an invalid ISBN (like "invalid-isbn!") would also be a good idea, so we can ensure that filterInvalid is working.

Reply:
I have added 979 ISBN and invalid isbn and updated the tests to test all variant more thoughtfully